OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -215,6 +228,16 @@ public virtual WVec GetRepulsionForce()
 			return dot >= 0 ? repulsionForce : WVec.Zero;
 		}
 
+		public void DisableRepulsing() {
```

Comment:
Oh wait. You still missed to put these braces on a new line.
```
public void DisableRepulsing()
{
```
please. Same for `EnableRepulsing` below.